rust-actix/src/database/models.rs

```diff
@@ -38,15 +37,14 @@ pub struct NewFollower<'a> {
     pub whom_id: &'a i32,
 }
 
-#[derive(Queryable, Selectable, Insertable)]
-#[diesel(table_name = crate::schema::messages)]
+#[derive(Queryable, QueryableByName, Selectable, Insertable)]
+#[diesel(table_name = crate::database::schema::messages)]
 #[diesel(check_for_backend(diesel::pg::Pg))]
-#[derive(QueryableByName)]
 pub struct Messages {
     pub message_id: i32,
     pub author_id: i32,
     pub text: String,
```

Comment:
I had issues with this is in Golang since it is called `text` in the database with key in the JSON object of the request is `content`. Just a hint if you run into the same issue

Reply:
I didn't understand the problem. Can you expand a bit the explanation? I think I had no problem here since it is running smoothly and passing through the tests.